blog/2025-07-29-admin-app-api-publisher-release-summary.md

```diff
@@ -0,0 +1,65 @@
+---
+title: '📢 Announcing! Ed-Fi Admin App v3.3.4 and API Publisher v1.3.0 Release'
+authors: hunter
+tags: [api-pub, admin-app, release]
+---
+
+Welcome to the July 2025 Ed-Fi release update! This post highlights the latest features, improvements, and fixes delivered in the Ed-Fi Admin App v3.3.4 and API Publisher v1.3.0. Whether you’re an implementer, developer, or community member, these updates are designed to make your Ed-Fi experience smoother and more secure.
+
+<!-- truncate -->
+
+## 📌 Key Updates
+
+### Ed-Fi Admin App v3.3.4
+
+- Accessibility & UX improvements (aria-hidden fixes, maxlength attributes, pagination, validation)
+- Bug fixes for modals, font loading, email validation, delete button, vendor management, and more
+- Security updates (input sanitization, dependency bumps, CodeQL)
+- Docker and developer experience enhancements
+
+### Ed-Fi API Publisher v1.3.0
+
+- New features: `authUrl` argument, namespace support for change version tracking
+- Improvements to dependency extraction and logging
+- Bug fixes for timeout, single change publishing, and remediation scripts
+- Security, documentation, and engineering automation updates
+
+## 🚀 Features Delivered So Far
+
+#### Ed-Fi Admin App
+
+- ODS instance and vendor management
+- Claim set and API client management
+- Support for SQL Server and PostgreSQL
+- Docker deployment for Windows and Linux
+- Accessibility and security improvements
+- UI enhancements and validation
+- Community-driven bug fixes and documentation
+
+#### Ed-Fi API Publisher
+
+- Change version tracking and publishing
+- Namespace-based publishing support
+- Authentication URL configuration
+- Dependency extraction from API metadata
+- Enhanced logging and diagnostics
+- Docker and automated build support
+- Security policy and engineering automation
+
+## 🛠️ Try It Out
+
+##### Ed-Fi Admin App
+
+See the [Admin App Documentation](https://docs.ed-fi.org/reference/admin-app) and [Docker deployment guide](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-AdminApp/blob/Docker/README.md) for installation and upgrade instructions.
```

Comment:
Removing the docs site base path, and linking to the Docker notes for this specific release.

```suggestion
See the [Admin App Documentation](/reference/admin-app) and [Docker deployment guide](https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-AdminApp/tree/v3.3.4/Docker) for installation and upgrade instructions.
```